plugins/net_plugin/net_plugin.cpp

```diff
@@ -3820,7 +3820,7 @@ namespace eosio {
            my->acceptor->bind(listen_endpoint);
            my->acceptor->listen();
          } catch (const std::exception& e) {
-           elog( "net_plugin::plugin_startup failed to bind to port ${port}", ("port", listen_endpoint.port()) );
+           elog( "net_plugin::plugin_startup failed to bind to port ${port}. Please check if the port is in use by another application. Either choose a different port or stop that application", ("port", listen_endpoint.port()) );
```

Comment:
This should be `fc_elog` instead of `elog`.
Add line wrap so code is easier to read.
I would recommend something more like: ".. . The port is likely in use by another application."

Reply:
will do, thanks Kevin!

Reply:
code change is in, thx